client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/InteractionCard.tsx

```diff
@@ -22,10 +21,14 @@ import PrivateHouseEventGrid from './PlacesAdditionalGrids/PrivateHouseEventGrid
 import OtherPublicLocationGrid from './PlacesAdditionalGrids/OtherPublicLocationGrid';
 import TransportationEventGrid from './PlacesAdditionalGrids/TransportationAdditionalGrids/TransportationEventGrid';
 
+import useStyles from './InteractionCardStyles';
+import useFormStyles from 'styles/formStyles';
```

Comment:
imports

Reply:
whats the problem here?

Reply:
import useStyles from './InteractionCardStyles'; in line 23
import useFormStyles from 'styles/formStyles'; with absolute paths imports

Reply:
but its more styles-related